spec/api-browser-window-spec.ts

```diff
@@ -6015,8 +6015,10 @@ describe('BrowserWindow module', () => {
           w.webContents.on('enter-html-full-screen', async () => {
             enterCount++;
             if (w.isFullScreen()) reject(new Error('w.isFullScreen should be false'));
-            const isFS = await w.webContents.executeJavaScript('!!document.fullscreenElement');
-            if (!isFS) reject(new Error('Document should have fullscreen element'));
+            await waitUntil(async () => {
+              const isFS = await w.webContents.executeJavaScript('!!document.fullscreenElement');
+              return isFS === true;
```

Comment:
Not a blocker, but asking in case I'm missing some nuance: is there a difference between `return isFS` and `return isFS === true`?

Reply:
@ckerr yes.  I had to do it that way because w.webContents.executeJavaScript returns `Promise<any>` but `waitUntil` expects a return of boolean.